myuw/templates/base.html

```diff
@@ -371,25 +385,146 @@ <h2 id="categories" class="sr-only">Categorized Resources</h2>
         {% include 'logger.html' %}
 
 
+        <!-- Opt-out Modal -->
+        <div id="optout_page-modal"></div>
+        <div class="myuw-modal modal fade" id="optout_modal" tabindex="-1" role="dialog" aria-labelledby="optout_modal_label" aria-hidden="true">
+            <div class="modal-dialog message-dialog" role="document">
+                <div class="modal-content">
+                    <div class="modal-header">
+                        <button id="optout_modal_close" type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                        <h3 class="modal-title" id="optout_modal_label">Switch to the Old Version of MyUW</h3>
+                    </div>
+                    <div class="modal-body">
+                        <br/>
+                        <p>Use the old version of MyUW as your default destination the next time you sign in.</p>
+
+                        <a href="/choose/legacy" class="btn btn-primary" title="Go to the old version of MyUW">Use the old version of MyUW</a> <button type="button" class="btn btn-default" data-dismiss="modal">Cancel</button>
+
+                        <br/><br/>
+                        <h4>Before you leave, share your feedback</h4>
+
+                        <p>We want to make MyUW work better for you. <a href="//myuw.washington.edu/cgi-bin/contactmyuw.cgi" target="_blank">Tell us why you're switching back.</a></p>
+                        <div class="myuw-ratings" style="text-align: center;">
+                            <div class="myuw-ratings-control"><!-- hide after rating is given -->
+                                <ol class="myuw-ratings-options">
+                                    <li><a class="opt-out-rate-myuw" rel="1-star" href="/choose/legacy" title="Rate MyUW 1 of 5 stars"><i class="fa fa-star-o" aria-hidden="true"></i><span class="sr-only">Rate MyUW 1 of 5 stars</span></a></li>
```

Comment:
Clicking a star should only submit the rating and hide/show the proper div here. It shouldn't automatically opt-out the user.